Comanche055/JET_SELECTION_LOGIC.agc

```diff
@@ -277,7 +277,7 @@ BDZCHECK	CA	ZNDX
 		EXTEND
 		BZF	TAUCHECK
 		
-ACRBDZ		CA	T5TEMP		# Z-TRANSLATION ACCEPTED EVEN THO WE MAY
+ACRBDZ		CA	T5TEMP		# Z-TRANSLATION ACCEPTED EVEN THOUGH WE MAY
```

Comment:
This is invalid. It's `THO` in the scans. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1044.jpg)

![](http://i.imgur.com/XLv4ZQ3.png)